src/components/TasksListItem/TasksListItem.tsx

```diff
@@ -124,36 +137,36 @@ export const TasksListItem = ({
         <div className={taskBodyClassName}>
           <div className='Layer__tasks-list-item__body-info'>
             <Text size={TextSize.sm}>{task.question}</Text>
-              <Textarea
-                value={userResponse}
-                placeholder={task.user_response_type === "UPLOAD_DOCUMENT" ? "Optional description" : ""}
-                onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) =>
-                  setUserResponse(e.target.value)
-                }
-              />
+            <Textarea
+              value={userResponse}
+              placeholder={task.user_response_type === 'UPLOAD_DOCUMENT' ? 'Optional description' : ''}
+              onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) =>
+                setUserResponse(e.target.value)
+              }
+            />
             {task.user_response_type === 'UPLOAD_DOCUMENT' ? (
-              <div className="Layer__tasks-list__link-list">
-              {selectedFiles ? (
-                <div className="Layer__tasks-list__link-list-header">Selected Files:</div>
-                ) : task.documents ? (
-                <div className="Layer__tasks-list__link-list-header">Uploaded Files:</div>
+              <div className='Layer__tasks-list__link-list'>
+                {selectedFiles ? (
+                  <div className='Layer__tasks-list__link-list-header'>Selected Files:</div>
+                ) : task.document ? (
+                  <div className='Layer__tasks-list__link-list-header'>Uploaded Files:</div>
                 ) : null}
-              <ul className="Layer__tasks-list__links-list">
-                {task.documents?.map((document, idx) => (
-                  <li key={`uploaded-doc-name-${idx}`}><a className="Layer__tasks-list-item__link" href={document.presigned_url.presignedUrl}>{document.file_name}</a></li>
-                ))}
-                {selectedFiles?.map((file, idx) => (
-                  <li key={`selected-file-name-${idx}`}><a className="Layer__tasks-list-item__link">{file.name}</a></li>
-                ))}
-              </ul>
+                <ul className='Layer__tasks-list__links-list'>
+                  {task?.document && Array.isArray(task.document) ? task.document.map((document, idx) => (
```

Comment:
Perhaps I'm connected to wrong API, but for response in case contains `document` field, not `documents`. Not sure which one is correct

Reply:
the new API (not published) includes `documents`. I have left `document` for backwards compatibility